tests/evaluate/test_evaluation.py

```diff
@@ -1365,3 +1365,97 @@ def test_evaluate_with_loaded_pyfunc_model():
     assert "mean_absolute_error" in run.data.metrics
     assert "mean_squared_error" in run.data.metrics
     assert "root_mean_squared_error" in run.data.metrics
+
+
+def test_evaluate_with_static_dataset_input_single_output():
```

Comment:
Can we add a test case for passing `mlflow.data.PandasDataset` as well?

Reply:
Added